CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Update controller container image to [`v0.48.1`](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v0.48.1). ([#211](https://github.com/giantswarm/nginx-ingress-controller-app/pull/211)). This release contains several performance improvements related to the admission webhook.
+- Breaking: Specify `--disable-svc-external-name` flag by default to disable forwarding to [ExternalName Services](https://kubernetes.io/docs/concepts/services-networking/service/#externalname). If you require this feature, you can disable setting this flag through user values by setting `controller.disableExternalNameForwarding` to `true`. ([#211](https://github.com/giantswarm/nginx-ingress-controller-app/pull/211))
```

Comment:
@pipo02mix Can you check the changelog entry if its clear enough? I'm planning to make the next release to be `2.0.0` to indicate the breaking change.